src/components/button-toggle/button-toggle.stories.tsx

```diff
@@ -2,27 +2,57 @@ import React from "react";
 import { ComponentStory } from "@storybook/react";
 
 import { ButtonToggle, ButtonToggleGroup } from ".";
-import Box from "../box";
 
 export const Default: ComponentStory<typeof ButtonToggle> = () => (
-  <ButtonToggleGroup id="button-toggle-group-id" label="Default example">
+  <ButtonToggleGroup
+    variant="default"
```

Comment:
**suggestion**: the `variant` prop doesn't need to be passed here

Reply:
@DobroTora just to state that we definitely don't need to be passing `variant="default"` here, your context approach has ensured that when no `variant` prop is present it will always default to `"default"`. If `variant="default"` was needed to be passed in order to ensure correct behaviour, you would have to do this on every story. Furthermore, this would be a massive issue for consumers as they'd have to opt back in to default functionality, causing huge visual changes to their UI. I've tested this locally with your branch and it looks spot on compared to master :). So yeah you're good to remove this imo